app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="form_container">
```

Comment:
- [ ] The login page should follow the layout from [sneak peek](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md), kindly style your pages accordingly

Your page as for now:
![Screenshot_20220930_011315](https://user-images.githubusercontent.com/25227022/193163638-219d9fe0-9e92-40e2-84b1-d9a6b62bec5b.png)